README.md

```diff
@@ -8,7 +8,39 @@ license](http://mit-license.org/).
 
 ## PRs for next issue are now being accepted
 
-Check the `drafts` directory for more details.
+To propose content for inclusion in the next newsletter (found in the `drafts/`
+folder), create a new [Pull Request][pr] updating the relevant section in the 
+draft.
+
+Alternately, tweet us [@thisweekinrust](https://twitter.com/thisweekinrust).
+
+### What do we look for when considering whether to include something in This Week in Rust?
+
+This Week in Rust is intended to highlight the incredible work of the Rust Community. 
+
+What we are generally looking for includes:
+
+* how-to intros (and advanced deep dives) into Rust concepts and areas
+* Rust walkthroughs
```

Comment:
Lately, I've been excluding basic walkthroughs of Rust concepts already covered by the Rustlings course, Book, or Rust by Example (ie, "here's what a trait looks like" or "Hello, Rust!!!"). Should we maybe mention something like that in the "what we aren't looking for"?

Reply:
I'm not sure if that's a good idea.
People have different styles to present, and people have different preferences how to consume.
I wouldn't want to see this excluded.

Reply:
I'd like to leave it in there - but maybe as editors we can focus on including things that may explain these concepts in different ways that what we have seen before?

Reply:
Actually - I think I know how to word this - adding it in.

Reply:
Done! Went with the wording "Rust walkthroughs that explain concepts in different ways than well known resources like [the Rust book](https://doc.rust-lang.org/stable/book/), [Rustlings](https://github.com/rust-lang/rustlings), and [Rust by Example](https://doc.rust-lang.org/stable/rust-by-example/)"